src/main/java/dev/meyi/bn/handlers/ChestTickHandler.java

```diff
@@ -36,6 +36,16 @@ public static void updateBazaarOrders(IInventory chest) {
     }
     items[items.length - 1] = Minecraft.getMinecraft().thePlayer.inventory.getItemStack();
 
+    Pattern p = Pattern.compile("(?:\\[.*\\] )(.*)");
+    Matcher m = p.matcher(StringUtils.stripControlCodes(Minecraft.getMinecraft().thePlayer.getDisplayName().getUnformattedText()));
+    String playerName;
+    if (m.find()) {
+      playerName = m.group(1);
+    } else {
+      System.err.println("Improperly formatted player name. Aborting!");
+      return;
```

Comment:
Instead of exiting this whole function when the player name is improperly formatted, can we just disable this feature so the rest of it still does the job.

Reply:
```suggestion
```